### ISSUE TITLE
Annotation Types are inconsistent for UWP vs android/IOS

### ISSUE THREAD

**Issue by leoek** [PSPDFKit/react-native#203]
# Details

I noticed that the annotation types have inconsistent names. Is there any reason for that?

This wouldn't be an issue if the wrapper would export these types as constants. Similar to the definitions here, which are uncommented for some reason: https://github.com/leoek/react-native-pspdfkit/blob/305b4d0fa2dcac76f6d01eea3761c564f6af36f5/ios/RCTPSPDFKit/Converters/RCTConvert%2BPSPDFConfiguration.m#L149

# Acceptance Criteria

- [ ] Export all annotation types with a consistent name (of the constant) though platforms.

# Example

I would like to import these types like this:

```
import { annotationTypes } from "react-native-pspdfkit"

const ink = annotatationTypes.ink;

/**
  * IOS/Android
  * ink being "Ink"
  *
  * UWP
  * ink being "ink"
  **/

```

**Comment by radazzouz** [comment]
@leoek, 

> I noticed that the annotation types have inconsistent names. Is there any reason for that?

Yes. The enums from that link are there for internal use between the Objective-C part of our React Native wrapper and our native SDK (PSPDFKit for iOS).

Those enums are not intended to be used from the React Native Javascript API.

Can you please tell me more about your use case and what is that you're trying to accomplish? So we can try to assist you with the existing Javascript API.

Thank you!

**Comment by leoek** [comment]
@radazzouz In our app, the documents and pdfs are handled by the app logic. The user is only allowed to annotate certain documents. Therefore I want to disable the annotations for some documents which are opened by the app for the user and restrict the possible types of annotations for some documents. (for example some documents shouldn't be annotated with images or sound).

Currently (for the UWP version) I have some configuration, for example:

```
allToolbarItems: [
      "sidebar-thumbnails",
      "sidebar-document-outline",
      "sidebar-annotations",
      "sidebar-bookmarks",
      "pager",
      "layout-config",
      "pan",
      "zoom-out",
      "zoom-in",
      "zoom-mode",
      "spacer",
      "annotate",
      "ink",
      "text-highlighter",
      "highlighter",
      "ink-signature",
      "image",
      "stamp",
      "note",
      "text",
      "line",
      "arrow",
      "rectangle",
      "ellipse",
      "polygon",
      "polyline",
      "print",
      "search"
    ],
    toolbarItemsWhiteList: {
      [BINARYTYPE.sometype]: [
        "sidebar-thumbnails",
        "sidebar-document-outline",
        "sidebar-annotations",
        "pager",
        "pan",
        "zoom-out",
        "zoom-in",
        "zoom-mode",
        "search"
      ]
    },
    toolbarItemsBlackList: {
      [BINARYTYPE.someothertype]: ["print", "image"]
    }
```

if a pdf is loaded I resolve the toolbarItems which should be available for that document and the apply them:

````
const setToolbarItemsPromise = this.refs.pdfView.setToolbarItems(
        toolbarItems
      );

**Comment by leoek** [comment]
@radazzouz  

> Those enums are not intended to be used from the React Native Javascript API.

It seems to work through the javascript api as described in the documentation. However I would like to import constants instead of defining different strings for windows and ios (and android?)

so I could use
```
editableAnnotationTypes: [annotationTypes.ink, "annotationTypes.highlight"];
```
instead of
```
editableAnnotationTypes: ["Ink", "Highlight"];
```
as described [here](https://github.com/PSPDFKit/react-native/blob/master/README.md#configuration-mapping)

**Comment by leoek** [comment]
@radazzouz any progress on this?

**Comment by radazzouz** [comment]
@leoek,

I'm sorry that we don't have any progress to report at the moment.

It would make sense to add an `annotationType` enum to be used for `editableAnnotationTypes`. We need to make this change on both Android and iOS simultaneously.

In the meantime, you could use the hardcoded strings for iOS, or create the enum in your own fork and then submit a pull request. We welcome and encourage pull request contributions from the community.
